tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -158,6 +158,7 @@ def _save_snapshot(self):
                             "Geomview", "VRML", "Wavefront", "Open Inventor",
                             "Povray", "X3D"]
 
+            wildcard = ""
```

Comment:
Why didn't tests catch this bug? Is there an opportunity to add a unit test here?